pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -0,0 +1,38 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc3-24126-00",
+    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24126-00",
```

Comment:
> we use the AppX package that contains in which we ship with the CoreRuntimeSDK.

Given this, what does it even mean to have CoreCLR nuget package in this manifest package?